### DISCUSSION TITLE
cannot pull llama3.1:latest

### DISCUSSION THREAD

**Discussion by airdogvan** [open-webui/open-webui#4506] [General]
open-webui version: v0.3.12
ollama version: ollama version is 0.3.4

OS: Ubuntu 22.04 LTS

When trying to dowload llama3.1 I get the following message:
```pull model manifest: 412: The model you are attempting to pull requires a newer version of Ollama. Please download the latest version at: https://ollama.com/download ```

Running docker.

Went in the ollama container and did: ollama run llama3.1:latest

It downloaded the model and ran it.  I could, from the cli, post queries that were correctly answered.

docker logs right after trying to download:
```open-webui    | []
open-webui    | INFO:     172.21.0.1:37260 - "GET /api/pipelines/list HTTP/1.1" 200 OK
open-webui    | INFO:     172.21.0.1:42444 - "GET /ollama/config HTTP/1.1" 200 OK
open-webui    | INFO:     172.21.0.1:42446 - "GET /ollama/api/version HTTP/1.1" 200 OK
open-webui    | INFO:     172.21.0.1:42444 - "GET /ollama/urls HTTP/1.1" 200 OK
open-webui    | INFO  [apps.ollama.main] url: http://host.docker.internal:11434
```

I checked again for other models and I can download Mixtral for instance...
Any help greatly appreciated.

**Comment by DaveLane-TS** [comment]
Are you using the [bundled Ollama support](https://github.com/open-webui/open-webui?tab=readme-ov-file#installing-open-webui-with-bundled-ollama-support)? I am and encountered the same issue. I resolved it just now by stopping everything, removing the ollama container, and pulling the latest before starting back up:

```sh
docker pull ghcr.io/open-webui/open-webui:ollama
```
